sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ReadFromKafkaDoFn.java

```diff
@@ -203,6 +205,8 @@ private ReadFromKafkaDoFn(ReadSourceDescriptors<K, V> transform) {
   @VisibleForTesting final DeserializerProvider<K> keyDeserializerProvider;
   @VisibleForTesting final DeserializerProvider<V> valueDeserializerProvider;
   @VisibleForTesting final Map<String, Object> consumerConfig;
+  @VisibleForTesting static final String METRIC_NAMESPACE = "KafkaIOReader";
+  @VisibleForTesting static final String RAW_SIZE_METRIC_PREFIX = "rawSize/";
```

Comment:
we shouldn't duplicate constant Strings like this. We can import them from KafkaUnboundedReader

Reply:
Done. Thanks for reviewing, @johnjcasey.